0-ad-history-mod-package/simulation/templates/units/han/infantry_crossbowman_b.xml

```diff
@@ -10,6 +10,12 @@
     <SelectionGroupName>units/han/infantry_crossbowman_b</SelectionGroupName>
     <SpecificName>Nǔ Shǒu</SpecificName>
     <Icon>units/han/infantry_crossbowman.png</Icon>
+    <History>
+        Although many ancient civilizations constructed crossbows, the Chinese were without a doubt the ones using them the most extensively. In the Han dynasty, crossbows were mass-produced and by far the most widely employed ranged weapon, especially among militias. The use of crossbows proved to be an important advantage in battle for the Han, most notably in the wars against the Xiognu, a nomadic horse people. The crossbow was a truly revolutionary weapon for several reasons.
+        Firstly, crossbows were very easy to use. It took years, if not an entire lifetime, to master the bow. On the other hand, even people who had never held a crossbow in their hands before could, with as little as a couple of days or weeks of training, learn to fight with the crossbow. Aiming, for example, was very easy; Han crossbows even had a scale built in, indicating how high to aim to hit the target at a specific distance. The low effort needed to train a good crossbowman was especially advantageous for the Han because their armies largely consisted of militias. When war broke out and the Han dynasty raised an army of peasants equipped with crossbows, each was as dangerous as a professional archer after years-long training.
+        Secondly, crossbows were cocked differently than bows. It was placed on the ground, and the bowman squatted down, grabbed the bowstring with his hands, and pulled it upwards. Because this movement utilizes arm, back, and leg muscles, more force gets exerted than through the motion of drawing a bow. This meant the crossbow could be tensed higher and fire arrows at a higher speed compared to bows. In fact, according to experiments, a Han crossbow required almost double the pulling force of a typical bow used by the Xiognu. But it also launched the arrow at 20 percent higher speed, up to 50 percent greater distance, and with a higher penetrating power. An arrow hitting its target caused dangerous, if not lethal, damage, even through the Xiognu's typical leather armor. In this regard, the Han crossbow was far superior to the Xiognu's bows.
```

Comment:
```suggestion
        Secondly, crossbows were drawn differently than bows. It was placed on the ground, and the bowman squatted down, grabbed the bowstring with his hands, and pulled it upwards. Because this movement utilizes arm, back, and leg muscles, more force gets exerted than through the motion of drawing a bow. This meant the crossbow could be tensed higher and fire arrows at a higher speed compared to bows. In fact, according to experiments, a Han crossbow required almost double the pulling force of a typical bow used by the Xiognu. But it also launched the arrow at 20 percent higher speed, up to 50 percent greater distance, and with a higher penetrating power. An arrow hitting its target caused dangerous, if not lethal, damage, even through the Xiognu's typical leather armor. In this regard, the Han crossbow was far superior to the Xiognu's bows.
```